flask_project/campaign_manager/test/test_intergration.py

```diff
@@ -0,0 +1,185 @@
+import os
+import unittest
+import ast
+
+from flask import abort, url_for, Flask
+from flask_testing import TestCase
+from geoalchemy2.shape import from_shape
+from shapely.geometry import asShape
+from datetime import date, datetime
+
+from campaign_manager.models.models import *
+from campaign_manager.sqlalchemy_session import engine, session, Base
+from campaign_manager.views import *
+from app import osm_app
+
+
+""" Geometry object for mock campaign """
+geom = {
+        "type": "Polygon",
+        "coordinates": [
+                [
+                    [
+                        -22.7271314710379,
+                        23.9871672761574
+                    ],
+                    [
+                        -22.7271314710379,
+                        23.9879122455024
+                    ],
+                    [
+                        -22.7264448255301,
+                        23.9879122455024
+                    ],
+                    [
+                        -22.7264448255301,
+                        23.9871672761574
+                    ],
+                    [
+                        -22.7271314710379,
+                        23.9871672761574
+                    ]
+                ]
+            ]
+        }
+
+
+class TestBase(TestCase):
+
```

Comment:
docstring for class. What does the TestBase class test?
Same for all the methods of the class.